File: Data Structures & Algorithms/two-integer-sum-ii/submission-2.java
class Solution {
    public int[] twoSum(int[] numbers, int target) {
        int n = numbers.length;
        for(int i = 1;i<n;i++){
            for(int j = i;j<n;j++){
                if(numbers[j] + numbers[j-i] == target){
                    return new int[]{j-i, j};
                }
            }
        }
        return new int[0];
    }
}

File: Data Structures & Algorithms/find-target-in-rotated-sorted-array/submission-2.py
class Solution:
    def search(self, nums: List[int], target: int) -> int:
        if len(nums) == 1:
            return target==nums[0]
        left,right=0,len(nums)-1

        while left<right:
            mid = left + (right-left)//2
            if nums[mid]==target:
                return mid
            elif nums[left]>target and nums[right]>target:
                left=mid+1
            else:
                right=mid
        return -1